spec/features/course/homepage_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Course: Homepage' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course, :opened) }
+    let(:course_student) { create(:course_student, :approved, course: course) }
+    let!(:valid_announcement) { create(:course_announcement, course: course) }
```

Comment:
@lowjoel uh but then I need to do 'let' twice  because there is two scenarios that need an announcement...